docs/installation.md

```diff
@@ -13,68 +13,82 @@ Currently, `jstz` can only be installed by building from sources using [Rust](ht
 Below are instruction on how to build `jstz` from source. Additionally, this section
 details how to install Octez, which is used for our local sandbox.
 
-### Cloning the Repository
+### Cloning the Repositories
+
+Get the source code for both `jstz` and Octez.
 
 ```sh
 git clone https://github.com/trilitech/jstz.git
+git clone https://gitlab.com/tezos/tezos.git
 ```
```

Comment:
You won't always have to build Octez from sources. 

Historically we did was due to floating point support and we only currently require sources since the `master` version of Octez is buggy

Reply:
That makes sense but should we keep it like this until we're out of the woods? Leaving building from source out of the docs as tribal knowledge would be harder for anyone trying to onboard to jstz.

Reply:
I'm not suggesting we leave it out, but cloning from sources for `tezos/tezos` is a step in installing Octez in the Octez documentation. 

We should avoid duplication between these two docs (only offering suggestions e.g. nix that might aid the user)